settings/prod.py

```diff
@@ -24,14 +24,19 @@
 
 INSTALLED_APPS += ('storages',)  # noqa
 
+AWS_STORAGE_BUCKET_NAME = "evalai"
```

Comment:
Would be good to have evalai/static bucket. Putting the static files in the top most hierarchy doesn't sound right.